lib/src/router/router_service_interface.dart

```diff
@@ -0,0 +1,7 @@
+import 'package:stacked/stacked.dart';
```

Comment:
@ferrarafer I was hoping this can go into the stacked_shared package but I see we need the RootStackRouter

Reply:
Yes, we need `RootStackRouter` and `RouteData` on the Interface.